Elixir Build.sublime-build

```diff
@@ -0,0 +1,11 @@
+{
+  "cmd": ["elixir", "${file}"],
+  "selector": "source.elixir",
+  "osx":{
+    "path": "/usr/local/bin/:${PATH}"
```

Comment:
Yes, so I am :-1:. A lot of folks use things like kiex or exenv. I personally use a custom directory and so on. :)